pep_sphinx_extensions/pep_theme/static/style.css

```diff
@@ -122,9 +122,6 @@ pre {
 dl dt {
     font-weight: bold;
 }
-dl dd {
-    margin: 0;
-}
 
```

Comment:
Let's fix the line spacing in a follow up.

Reply:
There are definitely other issues with line spacing, but this was one of the most immediately noticeable, and this particular issue is not just that the line spacing is too narrow but that is visibly inconsistent between definitions with multiple paragraphs and those with only a single one, so I'm not sure we should let the fact that there are also other issues with line spacing block fixing this one.

However, if we do this, this PR should be unmarked as resolving #2300 , because this was the initial problem that prompted and was cited in said issue.

Reply:
Opened as #2427